ingresses/staging/manager-staging-assets-ubuntu.yaml

```diff
@@ -0,0 +1,27 @@
+---
+
+kind: Ingress
+apiVersion: extensions/v1beta1  # See https://bit.ly/2KdOtrZ
+metadata:
+  name: manager-staging-assets-ubuntu
+  namespace: staging
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    nginx.ingress.kubernetes.io/ssl-redirect: "true"
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      more_set_headers "X-Robots-Tag: noindex";
```

Comment:
maybe this can be added for the prod configuration too. I don't think we want to index this project

Reply:
Yep, done!